src/api/app/controllers/webui/requests_listing_controller.rb

```diff
@@ -0,0 +1,41 @@
+class Webui::RequestsListingController < Webui::WebuiController
+  before_action :assign_attributes, :lockout_spiders
+  before_action :require_login
```

Comment:
```suggestion
  before_action :lockout_spiders
  before_action :require_login
  before_action :assign_attributes
```

We shouldn't assing any attributes if there isn't any user logged in...

Reply:
`assign_attributes` is just initializing URL. This action will be used in `project/package`. On these pages login is not required. We don't have to check login for it atm.

Reply:
Well, we haven't introduced `project/package` atm... These changes are related to something planned for the future, but not specifically to this commit.